payloads/assignpublicIP.go

```diff
@@ -26,37 +26,43 @@ type PublicIPCommand struct {
 	VnicMAC          string `yaml:"vnic_mac"`
 }
 
-// CommandAssignPublicIP is reserved for future use.
+// CommandAssignPublicIP wrapper of PublicIPCommand used during assigment of IP.
```

Comment:
`CommandAssignPublicIP is a wrapper around PublicIPCommand. It is the AssignPublicIP command payload`